src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -790,7 +790,7 @@ void PolicyHandler::OnSystemRequestReceived() const {
 }
 
 void PolicyHandler::TriggerPTUOnStartupIfRequired() {
-#ifndef EXTERNAL_PROPRIETARY_MODE
+#ifdef PROPRIETARY_MODE
```

Comment:
@iCollin The root cause of the defect was: `SDL does not start PTU sequence in HTTP mode` and you changed that it would start it only in `PROPRIETARY`. Is this correct?